bundles/org.openhab.binding.homematic/src/test/java/org/openhab/binding/homematic/internal/converter/ConvertFromBindingTest.java

```diff
@@ -75,28 +76,28 @@ public void testQuantityTypeConverter() throws ConverterException {
         floatQuantityDp.setUnit("°C");
         convertedState = temperatureConverter.convertFromBinding(floatQuantityDp);
         assertThat(convertedState, instanceOf(QuantityType.class));
-        assertThat(((QuantityType<?>) convertedState).getDimension(), is(UnitDimension.TEMPERATURE));
+        assertEquals(((QuantityType<?>) convertedState).getDimension(), SIUnits.CELSIUS.getDimension());
```

Comment:
Why change from constraint-based assert model, which is preferred here (`assertThat`)? Also, the parameters are now in wrong order.

Perhaps this is compatible:
```suggestion
        assertThat(((QuantityType<?>) convertedState).getDimension(), is(equalTo(SIUnits.CELSIUS.getDimension())));
```

Please check all.